_posts/2021-07-07-cockpit-248.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cockpit 248
+author: garrett
+date: '2021-07-07'
+tags: cockpit, machines, ostree
+slug: cockpit-248
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 248, Cockpit-Machines 247, and Cockpit-OSTree 187:
```

Comment:
...and cockpit-podman 32

Reply:
Oh. I guess it didn't have a release note either. I'll send up another (temporary) commit with the changes.